datastore/postgres/enrichment.go

```diff
@@ -106,7 +106,7 @@ VALUES
 ON CONFLICT
 DO
 	NOTHING;`
-		refreshView = `REFRESH MATERIALIZED VIEW latest_update_operations;`
+		refreshView = `REFRESH MATERIALIZED VIEW CONCURRENTLY latest_update_operations;`
```

Comment:
how often does this happen? Is this very resource intensive to do as often as it's done now?

Reply:
This should called ~ 400-500 times per update cycle (once per updater(rhel has a lot of updaters)). I don't think the frequency of this query is the issue, more the fact that it happens concurrently while we're sending 12000 matching queries / sec that reference this view.